lib/client/vm_api.rb

```diff
@@ -6,7 +6,11 @@ def vm(vm_id, opts={})
     end
 
     def vms(opts={})
-      headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      if opts[:without_details]
+        headers = {:accept => "application/xml"}
+      else
+        headers = {:accept => "application/xml; detail=disks; detail=nics; detail=hosts"}
+      end
       path = "/vms"
       path += search_url(opts) unless filtered_api
```

Comment:
@oourfali There is a [PR](https://github.com/theforeman/foreman/pull/3288) to allow setting the filtered API, which will soon be merged. 
That is why I ask - I do not want it to break the vms listing. 
How can we tell what info/options are available in the filtered api? It might also require other changes in our UI.